controllers/endpoint-controllers.js

```diff
@@ -46,3 +54,9 @@ exports.patchArticleVote = (request, response, next) => {
     .then(articleData => response.status(200).send({ upvotedArticle : articleData }))
     .catch(next)
 }
+
+exports.getUsers = (request, response, next) => {
+  selectUsers()
+    .then(usersData => response.status(200).send({ users: usersData }))
```

Comment:
this is brilliant, the only advise I can possibly give is that you can clean the code slightly by declaring the parameter as `users` instead of `userData` like so :

`.then(users => response.status(200).send({ users });`